test/spdmlib-test/src/common/util.rs

```diff
@@ -51,7 +51,12 @@ pub fn create_info() -> (SpdmConfigInfo, SpdmProvisionInfo) {
             | SpdmResponseCapabilityFlags::HBEAT_CAP
             | SpdmResponseCapabilityFlags::KEY_UPD_CAP
             | SpdmResponseCapabilityFlags::MUT_AUTH_CAP
-            | SpdmResponseCapabilityFlags::ENCAP_CAP,
+            | SpdmResponseCapabilityFlags::EP_INFO_CAP_NO_SIG
```

Comment:
Enable the capacity settings in unit tests.
Could change this configuration based on implementation in the future.